### ISSUE TITLE
[BUG]Password mismatch on new install

### ISSUE THREAD

**Issue by dudukee** [FreshRSS/FreshRSS#5953]
**Describe the bug：**

I'm gonna make this short. Exactly the same behavior as https://github.com/FreshRSS/FreshRSS/issues/5673#issuecomment-1838038920

I'm facing the same exact problem: "Password mismatch".

And i change the nginx config same with [this](https://github.com/FreshRSS/FreshRSS/tree/edge/Docker#hosted-as-domain-root)

Confusingly, the problem existed with **edge tag**, and after I **downgraded to freshrss/freshrss:1.22.1-arm**, the problem disappeared.

Environment information (please complete the following information):

Device: linux
OS: Ubuntu 20.04.3 LTS（aarch64）
Browser: MS Edge Latest
FreshRSS version: [docker edge](https://hub.docker.com/layers/freshrss/freshrss/edge/images/sha256-7c373800504aac348cb4eb8dee06592f0d32cd74713676c1f18a5213e63cd3e7)
Database version: integrated sqlite
PHP version: ?
Installation type: Docker

**Comment by Alkarex** [comment]
Could you please check your FreshRSS logs? See https://github.com/FreshRSS/FreshRSS/issues/5673#issuecomment-1838061469
Does the login work if you browse directly to your FreshRSS setup instead of through nginx?

**Comment by Dominion0815** [comment]
I also get the error with the current Edge image. With or without Nginx Proxy.

freshrss-app  | FreshRSS[54]: [dominion] [Tue, 19 Dec 2023 13:39:22 +0100] [warning] --- Password mismatch for user=dominion, nonce=.......

**Comment by dudukee** [comment]
I redeployed EDGE image with following log:
![CleanShot 2023-12-20 at 10 13 16](https://github.com/FreshRSS/FreshRSS/assets/3353717/62cdc3cd-0436-478c-845c-166cf1525734)

when I directly login whitout Nginx, it **can't work** properly either.

BUT with same Nginx config, the freshrss/freshrss:1.22.1-arm works  as expected

**Comment by Alkarex** [comment]
The empty `nonce` indicates that the session is not working. The problem can be both client side, at proxy level, or server side. Could you try with another browser?

**Comment by gazn3vada** [comment]
Same error here, docker edge arm installation, with or without nginx reverse proxy and tested with different browser (google chrome and firefox).
Logs with reverse proxy:

```
0" 302 - "-" "Mozilla/5.0 (X11; Linux x86_64; rv:120.0) Gecko/20100101 Firefox/120.0"
192.168.11.202 - - [20/Dec/2023:09:48:05 +0100] "GET /i/?rid=6582aa454f540 HTTP/1.0" 200 2328 "-" "Mozilla/5.0 (X11; Linux x86_64; rv:120.0) Gecko/20100101 Firefox/120.0"
192.168.11.202 - - [20/Dec/2023:09:48:05 +0100] "GET /themes/base-theme/frss.css?1702918781 HTTP/1.0" 200 8635 "https://cloud.xxxxxxxxx.com/freshrss/i/?rid=6582aa454f540" "Mozilla/5.0 (X11; Linux x86_64; rv:120.0) Gecko/20100101 Firefox/120.0"
192.168.11.202 - - [20/Dec/2023:09:48:05 +0100] "GET /themes/Origine/origine.css?1702918781 HTTP/1.0" 200 5212 "https://cloud.xxxxxxxxx.com/freshrss/i/?rid=6582aa454f540" "Mozilla/5.0 (X11; Linux x86_64; rv:120.0) Gecko/20100101 Firefox/120.0"
192.168.11.202 - - [20/Dec/2023:09:48:05 +0100] "GET /scripts/extra.js?1702918781 HTTP/1.0" 200 3033 "https://cloud.xxxxxxxxx.com/freshrss/i/?rid=6582aa454f540" "Mozilla/5.0 (X11; Linux x86_64; rv:120.0) Gecko/20100101 Firefox/120.0"
192.168.11.202 - - [20/Dec/2023:09:48:05 +0100] "GET /scripts/bcrypt.min.js?1702918781 HTTP/1.0" 200 11140 "https://cloud.xxxxxxxxx.com/freshrss/i/?rid=6582aa454f540" "Mozilla/5.0 (X11; Linux x86_64; rv:120.0) Gecko/20100101 Firefox/120.0"
192.168.11.202 - - [20/Dec/2023:09:48:05 +0100] "GET /scripts/main.js?1702918781 HTTP/1.0" 200 13396 "https://cloud.xxxxxxxxx.com/freshrss/i/?rid=6582aa454f540" "Mozilla/5.0 (X11; Linux x86_64; rv:120.0) Gecko/20100101 Firefox/120.0"
192.168.11.202 - - [20/Dec/2023:09:48:05 +0100] "GET /favicon.ico HTTP/1.0" 200 18102 "-" "Mozilla/5.0 (X11; Linux x86_64; rv:120.0) Gecko/20100101 Firefox/120.0"
192.168.11.202 - - [20/Dec/2023:09:48:05 +0100] "GET /themes/icons/favicon-256.png HTTP/1.0" 200 16450 "-" "Mozilla/5.0 (X11; Linux x86_64; rv:120.0) Gecko/20100101 Firefox/120.0"
192.168.11.202 - - [20/Dec/2023:09:48:05 +0100] "GET /themes/fonts/OpenSans.woff2 HTTP/1.0" 200 61980 "https://cloud.xxxxxxxxx.com/freshrss/themes/base-theme/frss.css?1702918781" "Mozilla/5.0 (X11; Linux x86_64; rv:120.0) Gecko/20100101 Firefox/120.0"
192.168.11.202 - - [20/Dec/2023:09:48:05 +0100] "GET /themes/icons/FreshRSS-logo.svg HTTP/1.0" 200 1950 "-" "Mozilla/5.0 (X11; Linux x86_64; rv:120.0) Gecko/20100101 Firefox/120.0"
192.168.11.202 - - [20/Dec/2023:09:48:05 +0100] "GET /themes/icons/key.svg HTTP/1.0" 200 353 "-" "Mozilla/5.0 (X11; Linux x86_64; rv:120.0) Gecko/20100101 Firefox/120.0"
192.168.11.202 - - [20/Dec/2023:09:48:05 +0100] "GET /themes/icons/login.svg HTTP/1.0" 200 328 "-" "Mozilla/5.0 (X11; Linux x86_64; rv:120.0) Gecko/20100101 Firefox/120.0"
192.168.11.202 - - [20/Dec/2023:09:48:05 +0100] "GET /themes/icons/close.svg HTTP/1.0" 200 360 "-" "Mozilla/5.0 (X11; Linux x86_64; rv:120.0) Gecko/20100101 Firefox/120.0"
192.168.11.202 - - [20/Dec/2023:09:48:08 +0100] "GET /i/?c=javascript&a=nonce&user=mikele HTTP/1.0" 200 112 "-" "Mozilla/5.0 (X11; Linux x86_64; rv:120.0) Gecko/20100101 Firefox/120.0"
FreshRSS[43]: [mikele] [Wed, 20 Dec 2023 09:48:08 +0100] [warning] --- Password mismatch for user=mikele, nonce=, c=$2a$04$rW2.***/mTClGDc2
192.168.11.202 - - [20/Dec/2023:09:48:08 +0100] "POST /i/?c=auth&a=login HTTP/1.0" 403 6163 "-" "Mozilla/5.0 (X11; Linux x86_64; rv:120.0) Gecko/20100101
```

no reverse proxy:

```
freshrss  | 192.168.11.202 - - [20/Dec/2023:09:52:32 +0100] "GET / HTTP/1.1" 302 - "-" "Mozilla/5.0 (X11; Linux x86_64) AppleWebKit/537.36 (KHTML, like Gecko) Chrome/120.0.0.0 Safari/537.36"
freshrss  | 192.168.11.202 - - [20/Dec/2023:09:52:33 +0100] "GET /i/?rid=6582ab5106db4 HTTP/1.1" 200 2331 "-" "Mozilla/5.0 (X11; Linux x86_64) AppleWebKit/537.36 (KHTML, like Gecko) Chrome/120.0.0.0 Safari/537.36"
freshrss  | 192.168.11.202 - - [20/Dec/2023:09:52:33 +0100] "GET /themes/base-theme/frss.css?1702918781 HTTP/1.1" 200 8635 "http://192.168.11.191:9999/i/?rid=6582ab5106db4" "Mozilla/5.0 (X11; Linux x86_64) AppleWebKit/537.36 (KHTML, like Gecko) Chrome/120.0.0.0 Safari/537.36"
freshrss  | 192.168.11.202 - - [20/Dec/2023:09:52:33 +0100] "GET /scripts/extra.js?1702918781 HTTP/1.1" 200 3033 "http://192.168.11.191:9999/i/?rid=6582ab5106db4" "Mozilla/5.0 (X11; Linux x86_64) AppleWebKit/537.36 (KHTML, like Gecko) Chrome/120.0.0.0 Safari/537.36"
freshrss  | 192.168.11.202 - - [20/Dec/2023:09:52:33 +0100] "GET /themes/Origine/origine.css?1702918781 HTTP/1.1" 200 5212 "http://192.168.11.191:9999/i/?rid=6582ab5106db4" "Mozilla/5.0 (X11; Linux x86_64) AppleWebKit/537.36 (KHTML, like Gecko) Chrome/120.0.0.0 Safari/537.36"
freshrss  | 192.168.11.202 - - [20/Dec/2023:09:52:33 +0100] "GET /scripts/bcrypt.min.js?1702918781 HTTP/1.1" 200 11140 "http://192.168.11.191:9999/i/?rid=6582ab5106db4" "Mozilla/5.0 (X11; Linux x86_64) AppleWebKit/537.36 (KHTML, like Gecko) Chrome/120.0.0.0 Safari/537.36"
freshrss  | 192.168.11.202 - - [20/Dec/2023:09:52:33 +0100] "GET /scripts/main.js?1702918781 HTTP/1.1" 200 13396 "http://192.168.11.191:9999/i/?rid=6582ab5106db4" "Mozilla/5.0 (X11; Linux x86_64) AppleWebKit/537.36 (KHTML, like Gecko) Chrome/120.0.0.0 Safari/537.36"
freshrss  | 192.168.11.202 - - [20/Dec/2023:09:52:33 +0100] "GET /themes/icons/login.svg HTTP/1.1" 200 328 "-" "Mozilla/5.0 (X11; Linux x86_64) AppleWebKit/537.36 (KHTML, like Gecko) Chrome/120.0.0.0 Safari/537.36"
freshrss  | 192.168.11.202 - - [20/Dec/2023:09:52:33 +0100] "GET /themes/icons/FreshRSS-logo.svg HTTP/1.1" 200 1950 "-" "Mozilla/5.0 (X11; Linux x86_64) AppleWebKit/537.36 (KHTML, like Gecko) Chrome/120.0.0.0 Safari/537.36"
freshrss  | 192.168.11.202 - - [20/Dec/2023:09:52:33 +0100] "GET /themes/icons/key.svg HTTP/1.1" 200 353 "-" "Mozilla/5.0 (X11; Linux x86_64) AppleWebKit/537.36 (KHTML, like Gecko) Chrome/120.0.0.0 Safari/537.36"
freshrss  | 192.168.11.202 - - [20/Dec/2023:09:52:33 +0100] "GET /themes/icons/close.svg HTTP/1.1" 200 360 "-" "Mozilla/5.0 (X11; Linux x86_64) AppleWebKit/537.36 (KHTML, like Gecko) Chrome/120.0.0.0 Safari/537.36"
freshrss  | 192.168.11.202 - - [20/Dec/2023:09:52:33 +0100] "GET /themes/fonts/OpenSans.woff2 HTTP/1.1" 200 61980 "http://192.168.11.191:9999/themes/base-theme/frss.css?1702918781" "Mozilla/5.0 (X11; Linux x86_64) AppleWebKit/537.36 (KHTML, like Gecko) Chrome/120.0.0.0 Safari/537.36"
freshrss  | 192.168.11.202 - - [20/Dec/2023:09:52:37 +0100] "GET /i/?c=javascript&a=nonce&user=mikele HTTP/1.1" 200 111 "-" "Mozilla/5.0 (X11; Linux x86_64) AppleWebKit/537.36 (KHTML, like Gecko) Chrome/120.0.0.0 Safari/537.36"
freshrss  | FreshRSS[42]: [mikele] [Wed, 20 Dec 2023 09:52:37 +0100] [warning] --- Password mismatch for user=mikele, nonce=, c=$2a$04$qNA41vIU3kr.***.dDxQPa
freshrss  | 192.168.11.202 - - [20/Dec/2023:09:52:37 +0100] "POST /i/?c=auth&a=login HTTP/1.1" 403 6163 "-" "Mozilla/5.0 (X11; Linux x86_64) AppleWebKit/537.36 (KHTML, like Gecko) Chrome/120.0.0.0 Safari/537.36"
freshrss  | 127.0.0.1 - - [20/Dec/2023:09:52:40 +0100] "OPTIONS * HTTP/1.0" 200 - "-" "Apache/2.4.57 (Debian) (internal dummy connection)"
freshrss  | 127.0.0.1 - - [20/Dec/2023:09:52:44 +0100] "OPTIONS * HTTP/1.0" 200 - "-" "Apache/2.4.57 (Debian) (internal dummy connection)"
```

**Comment by dudukee** [comment]
> The empty `nonce` indicates that the session is not working. The problem can be both client side, at proxy level, or server side. Could you try with another browser?

I can reproduce this problem with Chrome.

I don't think my environment that's causing the problem, because there is no problem after downgraded, but there is a problem with the Docker edge version.

**Comment by Alkarex** [comment]
Could you try with `freshrss/freshrss:edge-alpine` ?

**Comment by Alkarex** [comment]
Ok, bug found. Regression due to https://github.com/FreshRSS/FreshRSS/pull/5946

**Comment by Alkarex** [comment]
Should be fixed by https://github.com/FreshRSS/FreshRSS/pull/5955
New `edge` images ready in a few minutes. Please try again, and feedback welcome

**Comment by gazn3vada** [comment]
> Should be fixed by #5955 New `edge` images ready in a few minutes. Please try again, and feedback welcome

Fix works! Now login is ok with the last edge version.
Thanks!

**Comment by Alkarex** [comment]
Thanks for the feedback 👍🏻
